### DISCUSSION TITLE
Add products and change data as inventory and price

### DISCUSSION THREAD

**Discussion by pedromalito** [abuzuhri/Amazon-SP-API-CSharp#480] [Ideas]
I can with this code change price with api ? https://developer-docs.amazon.com/sp-api/docs/listings-items-api-v2021-08-01-reference , I don´t see this method

**Comment by TechyChap** [comment]
You can use the patch method to write the prices if you've already got the listing.

/attributes/purchasable_offer
/attributes/fulfillment_availability

https://developer-docs.amazon.com/sp-api/docs/listings-items-api-v2021-08-01-reference#patch-listings2021-08-01itemsselleridsku

**Comment by ayotade** [comment]
Could please help with the below, The code runs  just now revising the price @pedromalito @TechyChap @Paul-J-Barrett @shashi-attri 

`patch.listingsItemPatchRequest.patches.Add(new PatchOperation()
            {
                op = Op.replace,
                path = "/attributes/purchasable_offer",
                value = new object[] {  
                                        new{
                                                currency= "GBP",
                                                start_at=new {
                                                               value= DateTime.Now
                                                           },

                                                our_price=new []
                                                          { new{
                                                                    schedule =new[]
                                                              { new {
                                                                        value_with_tax= 36.02
                                                                    }
                                                              }}

                                                          }


                                             }
                                      }
            });`

Thank you

**Reply by ayotade** [reply]
It is actually working :) it just took some time for the price to update :)
